checks/automatic_dependency_update.go

```diff
@@ -20,16 +20,16 @@ import (
 	"github.com/ossf/scorecard/checker"
 )
 
-const check = "Automatic-Dependency-Update"
+const CheckDependencyUpdate = "Automatic-Dependency-Update"
```

Comment:
why not AutomaticDependencyUpdateCheck, reads better.

Reply:
The convention we've been using so far is to have check names start with `Check`. I'd vote for keeping the name same or probably making it `CheckAutomaticDependencyUpdate`.

Reply:
I am going with Azeem's suggestion to keep it consistent.  Thanks

Reply:
SGTM